source/agora/node/Ledger.d

```diff
@@ -760,7 +760,9 @@ public class Ledger
         if (auto fail_reason = this.getValidTXSet(data, tx_set))
             return fail_reason;
 
-        size_t active_enrollments = enroll_man.getValidatorCount(expect_height);
+        size_t active_enrollments = enroll_man.getValidatorCount(expect_height) -
+            data.missing_validators.length;
+        assert(active_enrollments >= 0);
```

Comment:
Please use a signed type if you're comparing like this.

Reply:
Updated, Ty.